integration/source.tf

```diff
@@ -58,6 +58,20 @@ resource "materialize_source_kafka" "example_source_kafka_format_text" {
   }
 }
 
+resource "materialize_source_kafka" "example_source_kafka_format_bytes" {
+  name = "source_kafka_bytes"
+  size = "2"
```

Comment:
This looks a bit funny, but it's also used for the avro source below. Is this actually working, or should this be `2xsmall`?